Rollbar/NetStandard/RollbarConfigurationLoader.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace Rollbar.NetStandard
+{
+    using System;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Reflection;
+
+    using Rollbar.Common;
+    using Rollbar.Telemetry;
+
+    /// <summary>
+    /// Class RollbarConfigurationLoader.
+    /// Implements the <see cref="Rollbar.NetStandard.IRollbarConfigurationLoader" />
+    /// </summary>
+    /// <seealso cref="Rollbar.NetStandard.IRollbarConfigurationLoader" />
+    public class RollbarConfigurationLoader 
+        : IRollbarConfigurationLoader
+    {
+        /// <summary>
+        /// The trace source
+        /// </summary>
+        private static readonly TraceSource traceSource = new TraceSource(typeof(RollbarConfigurationLoader).FullName);
+
+        /// <summary>
+        /// The known assembly names
+        /// </summary>
+        private readonly static string[] knownAssemblyNames = {
+            "Rollbar.App.Config",
+            "Rollbar.AppSettings.Json",
+            };
+
+        /// <summary>
+        /// The known loader type name
+        /// </summary>
+        private readonly static string knownLoaderTypeName = nameof(RollbarConfigurationLoader);
+
+        /// <summary>
+        /// The configuration loader type
+        /// </summary>
+        private readonly static Type configurationLoaderType = null;
+
+        /// <summary>
+        /// The loader
+        /// </summary>
+        private readonly IRollbarConfigurationLoader _loader = null;
+
+        /// <summary>
+        /// Initializes static members of the <see cref="RollbarConfigurationLoader" /> class.
+        /// </summary>
+        static RollbarConfigurationLoader()
+        {
+            string sdkAssembliesPath = RuntimeEnvironmentUtility.GetSdkRuntimeLocationPath();
+            traceSource.TraceInformation($"Rollbar SDK runtime location: {sdkAssembliesPath}");
+            Assembly assembly = null;
+            foreach(var name in RollbarConfigurationLoader.knownAssemblyNames)
+            {
+                string dllName = $"{name}.dll";
+                string dllFullPath = Path.Combine(sdkAssembliesPath, dllName);
+                if (!File.Exists(dllFullPath))
+                {
+                    continue;
```

Comment:
Nit: would it be helpful to emit a noisy log message here, in the event that we're expecting a DLL to be found but it's not?

Reply:
the alternative configuration loaders are completely optional. we use them only if we find them. there is no problem at all they are not found. it's all good!